examples/dev_flow/debug/debugging_visual.rst

```diff
@@ -1,16 +1,18 @@
-
 .. _examples_dev_flow_debug_visual:
 
 
 Debugging shared libraries with Visual Studio
 =============================================
 
-Conan packages don't contain the necessary information for debugging libraries with Visual Studio by default.
-This information is stored in PDBs that are generated during the compilation of the libraries. When using Conan these
-PDBs are generated in the build folder, which isn't always available, making impossible to debug the dependencies.
+We previously discussed how to debug dependencies in Conan with Visual Studio, but that requires the original build
```

Comment:
I think you should rephrase this, there's no such thing as `debug dependencies in Conan`